roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml

```diff
@@ -43,7 +43,7 @@ platforms:
     groups:
       - kafka_broker
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```

Comment:
Please remove this line completely, it's not necessary if we don't use the docker file.